### ISSUE TITLE
Colour/contrast issues

### ISSUE THREAD

**Issue by jvine** [sul-dlss/SearchWorks#537]
1. Disabled fonts are too light. They seem to be 0.65 opacity on a base colour of #ccc, which is nearly invisible. Base colour should be same as default #3f3c30, with opacity 0.75.

![screen shot 2014-07-21 at 2 23 20 pm](https://cloud.githubusercontent.com/assets/6945691/3650573/61b705c8-111d-11e4-849a-d314d1d4f252.png)
![screen shot 2014-07-21 at 2 23 07 pm](https://cloud.githubusercontent.com/assets/6945691/3650576/632ee006-111d-11e4-8d14-e5e6a1ccae4e.png)
1. Select on record view toolbar should be red like the other controls.

![screen shot 2014-07-21 at 2 24 04 pm](https://cloud.githubusercontent.com/assets/6945691/3650582/71c56ec8-111d-11e4-97fd-375a4d2b2743.png)
![screen shot 2014-07-21 at 2 23 50 pm](https://cloud.githubusercontent.com/assets/6945691/3650584/73a96078-111d-11e4-83fa-2b14c1638512.png)
1. Date in results is #999, should be #767676.

![screen shot 2014-07-21 at 2 21 07 pm](https://cloud.githubusercontent.com/assets/6945691/3650521/00c3b220-111d-11e4-8924-79c33d5f27f6.png)
![screen shot 2014-07-21 at 2 21 20 pm](https://cloud.githubusercontent.com/assets/6945691/3650522/02dce752-111d-11e4-9b55-7d8fd4495dae.png)
1. Active page in search results pagination should match the Database A-Z selector: background colour is currently #428bca and should be dimgrey.

![screen shot 2014-07-21 at 2 18 07 pm](https://cloud.githubusercontent.com/assets/6945691/3650511/e49f80b0-111c-11e4-8b7d-84c19e1012b8.png)
![screen shot 2014-07-21 at 2 18 24 pm](https://cloud.githubusercontent.com/assets/6945691/3650510/e49c88d8-111c-11e4-80cb-0966359e8ddc.png)
